### DISCUSSION TITLE
How do I setup synchronous transforming for vanilla js/ts files?

### DISCUSSION THREAD

**Discussion by ghost** [swc-project/swc#4508] [Q&A]
# SWC Documentation needs to have more examples.

Hello I need to see an **example** how that is set up as the documentation is still in need of further explanation and I don't have all the bits to figure it out as I am new to the whole compilation/bundling/minification topics.

I would really love it if someone posts a repo how they implemented synchronous transforming with swc/wasm so that I can get to learn how to properly speed up the whole process otherwise I don't get the desired results 😄  

Thanks in advance 🔥

**Comment by kdy1** [comment]
There are lots of example in the repo, if you are using it from rust

**Reply by ghost** [reply]
The issue is I don't know rust yet so I was wondering how do I use it from a vanilla js perspective to make it extremely fast @kdy1

**Reply by ghost** [reply]
Also can u send me a link to an example that used swc wasm for speeding it all up? :)

**Reply by daflyinbed** [reply]
https://github.com/swc-project/swc-playground/blob/2b89e07096210d9454d6c31c49f11bedcde88631/src/swc.ts#L276
[play.swc.rs](https://play.swc.rs/) is using swc wasm.

**Reply by ghost** [reply]
Do we have a way of using it via the config file or inside of webpack ideally? @daflyinbed

**Reply by RiESAEX** [reply]
webpack: https://github.com/swc-project/swc-loader

**Reply by ghost** [reply]
I know about that but the question is how do I get the 20x ~ 70x of speed compared to babel? 😄 @RiESAEX

**Reply by RiESAEX** [reply]
https://github.com/swc-project/website/blob/175def633571c164131a25ddb7e39b79b2927007/blog/2020-02-16-why-is-swc-fast.md
This old blog which has been removed in a refactor of website talks about why swc is fast

**Reply by ghost** [reply]
So essentially here is no way to enable that speed out of the box with an option in the swcrc or webpack to do the 70 times faster operation? @RiESAEX
